README.md

```diff
@@ -15,21 +15,21 @@ Download or clone this repo and open the `csharp-101` folder in VS Code to get s
+2  | The Basics of Strings         | [02 Notebook](https://tinyurl.com/csharp101-notebook02-ipynb) | [02 Video](https://www.youtube.com/watch?v=JSpC7Cz64h0&list=PLdo4fOcmZ0oVxKLQCHpiUWun7vlJJvUiN&index=3) | [Intro to C#](https://docs.microsoft.com/dotnet/csharp/tour-of-csharp/tutorials/hello-world?WT.mc_id=csharpnotebook-35129-website)
+3  | Searching Strings             | [03 Notebook](https://tinyurl.com/csharp101-notebook03-ipynb) | [03 Video](https://www.youtube.com/watch?v=JL30gSE3WaQ&list=PLdo4fOcmZ0oVxKLQCHpiUWun7vlJJvUiN&index=4) | [Intro to C#](https://docs.microsoft.com/dotnet/csharp/tour-of-csharp/tutorials/hello-world?WT.mc_id=csharpnotebook-35129-website)
+4  | Numbers and Integers Math     | [04 Notebook](https://tinyurl.com/csharp101-notebook04-ipynb) | [04 Video](https://www.youtube.com/watch?v=jEE0pWTq54U&list=PLdo4fOcmZ0oVxKLQCHpiUWun7vlJJvUiN&index=5) | [Numbers in C#](https://docs.microsoft.com/dotnet/csharp/tour-of-csharp/tutorials/numbers-in-csharp?WT.mc_id=csharpnotebook-35129-website)
+5  | Numbers and Integer Precision | [05 Notebook](https://tinyurl.com/csharp101-notebook05-ipynb) | [05 Video](https://www.youtube.com/watch?v=31EmPADtv4w&list=PLdo4fOcmZ0oVxKLQCHpiUWun7vlJJvUiN&index=6) | [Numbers in C#](https://docs.microsoft.com/dotnet/csharp/tour-of-csharp/tutorials/numbers-in-csharp?WT.mc_id=csharpnotebook-35129-website)
+6  | Numbers and Decimals          | [06 Notebook](https://tinyurl.com/csharp101-notebook06-ipynb) | [06 Video](https://www.youtube.com/watch?v=kdKcpF9roeU&list=PLdo4fOcmZ0oVxKLQCHpiUWun7vlJJvUiN&index=7) | [Numbers in C#](https://docs.microsoft.com/dotnet/csharp/tour-of-csharp/tutorials/numbers-in-csharp?WT.mc_id=csharpnotebook-35129-website)
+7  | Branches (if)                 | [07 Notebook](https://tinyurl.com/csharp101-notebook07-ipynb) | [07 Video](https://www.youtube.com/watch?v=y4OTe8LSokg&list=PLdo4fOcmZ0oVxKLQCHpiUWun7vlJJvUiN&index=8) | [Branches and Loops in C#](https://docs.microsoft.com/dotnet/csharp/tour-of-csharp/tutorials/branches-and-loops-local?WT.mc_id=csharpnotebook-35129-website)
+8  | What Are Loops?               | [08 Notebook](https://tinyurl.com/csharp101-notebook08-ipynb) | [08 Video](https://www.youtube.com/watch?v=z31m5Up_gSQ&list=PLdo4fOcmZ0oVxKLQCHpiUWun7vlJJvUiN&index=10) | [Branches and Loops in C#](https://docs.microsoft.com/dotnet/csharp/tour-of-csharp/tutorials/branches-and-loops-local?WT.mc_id=csharpnotebook-35129-website)
+9  | Combining Branches and Loops  | [09 Notebook](https://tinyurl.com/csharp101-notebook09-ipynb) | [09 Video](https://www.youtube.com/watch?v=qK7tUpaOXi8&list=PLdo4fOcmZ0oVxKLQCHpiUWun7vlJJvUiN&index=11) | [Branches and Loops in C#](https://docs.microsoft.com/dotnet/csharp/tour-of-csharp/tutorials/branches-and-loops-local?WT.mc_id=csharpnotebook-35129-website)
```

Comment:
Notebook 9 Doesn't work

Reply:
@jamesmontemagno the url used by the tinyurl is wrong for 2 and 9.

Reply:
More context:
https://github.com/dotnet/interactive/issues/1861#issuecomment-1027196647